hq/public/stylesheets/main.css

```diff
@@ -302,6 +301,10 @@ table.iam-report__table th {
   box-sizing: initial;
 }
 
+.iam-modal__trigger {
+  height: 24px;
```

Comment:
Make sure MaterializeCSS can't already do what you're after here.

http://materializecss.com/icons.html